### ISSUE TITLE
Add widget for textual entailment/NLI

### ISSUE THREAD

**Issue by lighteternal** [huggingface/huggingface_hub#359]
**Is your feature request related to a problem? Please describe.**
The textual entailment / NLI task cannot be easily visualized through the Hosted inference API. Widget pipeline tags exist for most downstream tasks (e.g. 0-shot classification, QA, STS)

**Describe the solution you'd like**
A widget dedicated to textual entailment (e.g. outputting whether the hypothesis entails/contradicts/is neutral to the premise would be really nice.

**Comment by julien-c** [comment]
so it would be two text inputs + output is labels classification, is that correct?

**Comment by lighteternal** [comment]
Exactly!  A pair of sentences as input, and one of three labels (contradiction, entailment, neutral) as output. 

An example that could benefit from this widget can be found at the Usage section of this model card: https://huggingface.co/cross-encoder/nli-deberta-base
